dotnet/src/SemanticKernel/TemplateEngine/PromptTemplateEngine.cs

```diff
@@ -105,20 +111,31 @@ public IList<Block> RenderVariables(IList<Block> blocks, ContextVariables? varia
         SKContext executionContext)
```

Comment:
As far as I can tell, there's nobody using this method. It also doesn't seem like a super useful method to be part of the public SDK surface. @dehoward Instead of updating this method, could you please just remove it?
(Also remove it from the interface, IPromptTemplateEngine, and any unit tests that hit this method directly).